Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -73,14 +73,14 @@
         }
 
         tileset.style = new Cesium.Cesium3DTileStyle(tileset, {
-//            "color" : [255, 255, 255]
+//            "color" : "#ff00ff",
```

Comment:
Can you add examples here for all the CSS color format variants?  Then, we'll use these for unit tests starting on Friday.